_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -25086,14 +25086,14 @@
 	name = "Queue Shutters Control";
 	pixel_x = 25;
 	pixel_y = -36;
-	req_access_txt = "28"
+	req_access = list(57)
```

Comment:
bloons3 is correct: this should be set to req_access_txt = "57" for everything you've edited